### PR TITLE
Add new board: Cytron Maker UNO RP2040

### PR DESCRIPTION
**Pull request by noqman**
Hi,

Can you please help to add this new board : Cytron Maker UNO RP2040 ?

Thanks

### CODE REVIEW

File: ports/raspberrypi/boards/cytron_maker_uno_rp2040/pins.c [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+
+    { MP_ROM_QSTR(MP_QSTR_GP20), MP_ROM_PTR(&pin_GPIO20) },
+    { MP_ROM_QSTR(MP_QSTR_D20), MP_ROM_PTR(&pin_GPIO20) },
+    { MP_ROM_QSTR(MP_QSTR_LED20), MP_ROM_PTR(&pin_GPI20) },
+    { MP_ROM_QSTR(MP_QSTR_SDA), MP_ROM_PTR(&pin_GPIO20) },
+
+    { MP_ROM_QSTR(MP_QSTR_GP21), MP_ROM_PTR(&pin_GPIO21) },
+    { MP_ROM_QSTR(MP_QSTR_D21), MP_ROM_PTR(&pin_GPIO21) },
+    { MP_ROM_QSTR(MP_QSTR_LED21), MP_ROM_PTR(&pin_GPI21) },
```
Comment (dhalbert) [inline ports/raspberrypi/boards/cytron_maker_uno_rp2040/pins.c:92]: Typo
```suggestion
    { MP_ROM_QSTR(MP_QSTR_LED21), MP_ROM_PTR(&pin_GPIO21) },
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Hi - one typo, and pre-commit says there are blank lines with trailing spaces in `pins.c`. See https://learn.adafruit.com/building-circuitpython/build-circuitpython#install-pre-commit-3096511 for more information about using pre-commit locally to catch these kinds of things.

You can also set some editors to automatically remove trailing spaces and trailing blank lines.

**Comment by dhalbert** [comment]
Hi @noqman -
There are still two problems with the formatting. See https://github.com/adafruit/circuitpython/actions/runs/6967019048/job/18958156375?pr=8649
1. `mpconfigboard.mk` needs a single newline at the end.
2. There are lines that consist solely of spaces in `pins.c`.

I would fix these for you but you have not given permission for maintainers to push to your PR branch.

**Comment by dhalbert** [comment]
I'm not sure what happened. Your commit f2b39ae contained some extraneous changes unrelated to to your new board. Merging from upstream did not fix that. If you `git reset --hard` back to 6d02f12 and then just fix `mpconfigboard.mk` and `pins.c` and `git push -f`, I think it will be OK.

**Comment by noqman** [comment]
Hi, I have implemented all your suggestions. Please let me know if any further revisions or changes are needed. 

Thank you and I truly appreciate your guidance.

**Review by dhalbert — APPROVED** [review]
OK, this looks fine. Thanks for your patience with the formatting changes!

**Pull request merged**